db/migrate/20180810162637_delete_assignments_with_duplicate_mission_and_user.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class DeleteAssignmentsWithDuplicateMissionAndUser < ActiveRecord::Migration[5.1]
+  def up
+    duplicates = []
+    dup_attrs = Assignment.select(:mission_id, :user_id).group(:mission_id, :user_id).having("count(*) > 1")
+
+    dup_attrs.each do |dup|
+      dup_assignments = Assignment.where(mission_id: dup.mission_id, user_id: dup.user_id)
+      duplicates << dup_assignments.first
```

Comment:
This is close! But it assumes there are only two duplicates. What if there are 50?

You can grab _all but the first_ elements in an array with `array[1..-1]`. Note that you'll then have an array, not a single object.